enterprise/models.py

```diff
@@ -961,6 +961,22 @@ def get_course_run(self, course_run_id):
             return None
         return CourseCatalogApiServiceClient(self.enterprise_customer.site).get_course_run(course_run_id)
 
+    def get_course_and_course_run(self, course_run_id):
+        """
+        Get course data and all of the metadata for the given course run.
+
+        Arguments:
+            course_run_id (str): The course run key which identifies the course run.
+
+        Returns:
+            tuple(course, course_run): The course and course run metadata.
+
+        """
+        if not self.contains_content_items('key', [course_run_id]):
+            return None, None
+
+        return CourseCatalogApiServiceClient(self.enterprise_customer.site).get_course_and_course_run(course_run_id)
```

Comment:
In case this needs to be handled gracefully, would need a try-catch block for `ImproperlyConfigured`.

Reply:
We don't need to hide any error related to configuration. I will add `Raises` section in the docstring about the exception `ImproperlyConfigured`.

Reply:
Done